test/Concurrency/derived_conformances_nonisolated.swift

```diff
@@ -10,14 +10,11 @@ struct X1: Equatable, Hashable, Codable {
   let y: String
 }
 
-// expected-error@+5 3{{main actor-isolated property 'y' can not be referenced from a non-isolated context}}
-// expected-note@+4{{in static method '==' for derived conformance to 'Equatable'}}
-// expected-error@+3{{main actor-isolated property 'y' can not be referenced from a non-isolated context}}
-// expected-note@+2{{in static method '==' for derived conformance to 'Equatable'}}
+// okay
```

Comment:
Sweet! Could you please add another test that has non-Sendable property types where these diagnostics still show up? I believe I added this test to make sure the diagnostics in synthesized conformance implementations still have a valid source location.

Reply:
Hmm, with non-Sendable types, there's a different kind of error I'm seeing: `cannot automatically synthesize 'Encodable' because 'NonSendable' does not conform to 'Encodable'`